src/main/java/io/confluent/kafkarest/SimpleConsumerManager.java

```diff
@@ -64,13 +49,13 @@
   private final MetadataObserver mdObserver;
   private final SimpleConsumerFactory simpleConsumerFactory;
```

Comment:
By the way, it seems rather excessive to have `SimpleConsumerManager`, `SimpleConsumerPool`,  and a `SimpleConsumerFactory`. Do you think it would make sense to at least consolidate `SimpleConsumerPoll` and `SimpleConsumerManager` into a single class or do you think it's better to keep the separation?